.github/workflows/tests.yaml

```diff
@@ -15,7 +15,7 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-go@v2
         with:
-          go-version: 1.19.1
+          go-version: 1.19.3
```

Comment:
Looks like go-version is being set on line 51 as well.

Reply:
:+1: